test/component/db_test.js

```diff
@@ -27,15 +27,19 @@ var should = require('should'),
     utilsT = require('../utils/utilsT'),
     db = require('../../lib/db');
 
-describe('Db', function() {
+xdescribe('Db', function() {
```

Comment:
Disabled while https://stackoverflow.com/questions/72699235/mongodb-node-driver-connect-ignores-connecttimeoutms-and-sockettimeoutms-set gets clarified

Reply:
Question was answered. Based on it, fixed in 5be543e08f5125df3ea8ca9549cd29eb95277556